server/index.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import users from './routes/users';
+import boards from './routes/boards';
+
+const app = express();
+const port = process.env.PORT || 3000;
```

Comment:
Create a constants file and use it from there, so that you have a list of environment variables at one place

Reply:
Done